django/contrib/auth/hashers.py

```diff
@@ -21,13 +21,8 @@
 
 
 def is_password_usable(encoded):
-    if encoded is None or encoded.startswith(UNUSABLE_PASSWORD_PREFIX):
-        return False
-    try:
-        identify_hasher(encoded)
-    except ValueError:
-        return False
-    return True
+    """Is the password something besides make_password(None)?"""
+    return encoded is None or not encoded.startswith(UNUSABLE_PASSWORD_PREFIX)
```

Comment:
`None` is supposed to be a usable password?

Reply:
Looking at 703c266682be39f7153498ad0d8031231f12ee79 again, I see that `None` was previously treated as a disabled password. There were no tests or documentation about the rationale for that however. 

Claude, https://github.com/django/django/pull/9276 proposed a separate method as you suggested, however, I think that would leave `is_usable_password()` unused by Django as the two places it's used, I believe we want the semantics of "was `set_unusable_password()` called?" I don't see a use case for the current semantics, besides in `check_password()` which is adapted here. I updated the documentation for `User.has_usable_password()` only a couple months ago in (92f48680dbd2e02f2b33f6ad0e35b7d337889fb2 when I was looking at this issue previously and I thought a separate API was needed, but I don't think your change in 703c266682be39f7153498ad0d8031231f12ee79 was entirely intentional as you didn't update the documentation at that time. Feel free to elaborate on your thinking.

Reply:
Sure, I've added that.